CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to copilot-markdown-generator
+
+## Code Contributions
+
+### Requirements
+
+You will need to have installed on your machine:
+
+* PHP (v5.6.0 or above). If you're using macOS, you already have this installed.
+* Composer. [Get Composer][Composer]
+
+There are useful links in the [see also][CONTRIBUTING see also]
+section below, as well as in the [readme][README see also].
+
+### Step 1: Fork
+
+Fork (or branch) the project [on GitHub][GitHub repo] and clone it.
+
+```bash
+$ git clone git@github.com:username/copilot-markdown-generator-php.git
+$ git remote add upstream https://github.com/conde-nast-international/copilot-markdown-generator-php.git
+```
+
+### Step 2: Install
+
+Install project dependencies to `vendor/`.
```

Comment:
This line confused me as I wasn't familiar enough with composer to know that it would be installing the deps in vendor rather than it being something separate I had to do before running `composer install`

Reply:
@nigelhanlon ohh I see. I'll make this a colon if that's clearer.

Reply:
Yeah that would make it clearer 👍